### DISCUSSION TITLE
Good things to add in the next update?

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#47] [Ideas]
- portal protection radius (per arena)
- a map selection gui for duel request if has permission
- pls make 20 nodamage ticks default I keep forgetting to change 19 to 20 and then the kb is dudu (optional)
- make some options like snowball break blocks and gapple effects default true
- all arena clones are made in a void world (cuz trees generate on copies due to wdl settings)
- some free spigots like windspigot support

**Comment by HierarchyMan** [comment]
@GoodestEnglish

**Comment by GoodestEnglish** [comment]
portal protection radius per arena is coming soon
map selection gui is already exists, by default it is disabled, user can toggle it in /settings. The permission of the settings is `eden.settings.arena-selection`
most of the server damage ticks is 19, this default value will not be changed. 
snowball break blocks already exists, it is in config.yml
tree should not be generated or grown. It is disabled by Eden by default. I am not sure what is wdl means
We are def adding more spigot support soon. By as I mentioned before, I would like to focus on bug fixes right now.

**Comment by HierarchyMan** [comment]
I meant some options should be enabled by default for qol
And yes the trees don't generate when ur playing in clone arena but when when you tp to it and move around trees generate

**Comment by GoodestEnglish** [comment]
By default those are disabled because some servers wish to give specific features to their donors, like map selection.
I will need more verify on the tree thing, because it is already cancelled by Eden in [here](https://github.com/diamond-rip/Eden/blob/master/src/main/java/rip/diamond/practice/misc/listeners/GeneralListener.java#L108)

**Comment by HierarchyMan** [comment]
Ok the trees thing worked
I generated the world on my own before uploading it to the server which caused issues
portal prot per arena is much requested cuz ur a skilled coder
And this is the best plugin
